experimental/table/server/under/hive/src/main/java/alluxio/table/under/hive/HiveDatabase.java

```diff
@@ -181,6 +183,8 @@ public UdbTable getTable(String tableName) throws IOException {
           HiveUtils.toProtoSchema(table.getAllCols()), tableUri.getPath(),
           Collections.singletonMap("unpartitioned", builder.build()),
           HiveUtils.toProto(table.getPartitionKeys()), partitions, table);
+    } catch (InvalidTableException e) {
+      throw new NotFoundException("Table " + tableName + " not found", e);
```

Comment:
Could we make the exception messages consistent as well as use `String.format` or just String concatenation consistently?

Reply:
fixed. using String concatenation only.